jax/_src/scipy/signal.py

```diff
@@ -33,6 +33,68 @@
 from jax._src.typing import Array, ArrayLike
 from jax._src.util import canonicalize_axis, tuple_delete, tuple_insert
 
+@_wraps(osp_signal.fftconvolve)
+def fftconvolve(in1: Array ,in2: Array, mode: str ="same",axes=None) -> Array:
```

Comment:
Inputs should have type `ArrayLike`; see https://jax.readthedocs.io/en/latest/jax.typing.html#jax-typing-best-practices

Reply:
done